tests/apps_test.go

```diff
@@ -175,14 +175,11 @@ var _ = Describe("deis apps", func() {
 				Eventually(sess).Should(Exit(0))
 			})
 
-			// Pending: The logging subsystem is not currently installed as part of the workflow-dev
-			// chart.
-			XSpecify("that user can retrieve logs for that app", func() {
+			Specify("that user can retrieve logs for that app", func() {
 				sess, err := cmd.Start("deis logs -a %s", &user, app.Name)
```

Comment:
Nevermind, after talking privately I understand that this runs the command as the given used, with `app.Name` as the single param to the format string